bixlabs/authentication
# implemented the system storage interface in sqlite

## Description 
SQLite implementation for the system to have a database storage instead of a memory one.

## Stories
https://bixlabs.atlassian.net/browse/AM-19

## List of changes 
* Implementation of the whole storage interface with SQLite.
* Minor changes around memory storage, tests were breaking weridly because of a bad implementation of it.

## Steps to Test or Reproduce
* Run `make run-cli` and see how the SQLite database is created, take that file and find a SQLite database viewer (I used IntellijIdea for this), notice that the information that's added in the main it's in the database.

## Impacted Areas in Application 
List general components of the application that this PR will affect.
* Storage / Database

## Migrations
NO
